src/sass/_Fabric.Utilities.scss

```diff
@@ -34,3 +34,38 @@
   @include resetPadding;
   @include resetBoxShadow;
 }
+
+// To set various text alignment styles 
+// @param [variable list] $alignments
+@mixin ms-u-textAlign($alignments...) {
+  @each $align in $alignments {
+    .ms-u-textAlign-#{$align} {
+      text-align: $align;
+    }
+  }
+}
+
+// To hide content while still making it readable by screen readers (Accessibility)
+@mixin ms-u-sr-only {
```

Comment:
Mixin and class name should be camelCase instead of dashed. E.g. `ms-u-srOnly` or `ms-u-screenReaderOnly`